Add context to rdfa-visualiser to say if we're at the top level

piemonkey opened the pull request:
### Overview
Allows for hiding elements only when they are not part of a hierarchy for example, snippets.

##### connected issues and PRs:
Related to https://github.com/lblod/ember-rdfa-editor/pull/1333

### Setup
N/A

### How to test/reproduce
In the editable-nodes sample, Snippets should be hidden from the top level, so pasting in HTML with snippets should not display them in the top level (even if the nodespecs don't work for obvious reasons). Linking any resource to the snippet should allow it to appear in the visualiser in the correct place.

### Challenges/uncertainties
N/A

### Checks PR readiness
- [ ] UI: works on smaller screen sizes
- [ ] UI: feedback for any loading/error states
- [ ] Check cancel/go-back flows
- [ ] Check database state correct when deleting/updating (especially regarding relationships)
- [x] changelog
- [x] npm lint
- [x] no new deprecations

elpoelma reviewed (changes requested):
https://github.com/lblod/ember-rdfa-editor/blob/09451449a9bcde555ea2c331c5672f2e7c552948/packages/ember-rdfa-editor/src/components/_private/relationship-editor/card.gts#L444 gives type errors now, as the `@isTopLevel` argument is not passed. 
Alternatively we could also not pass the `isTopLevel` arg to the `configurable-rdfa-display` component, and already encode it in the generator function we pass to that component. (as the component itself doesn't really need that argument, and it is not super relevant for every usage of it)

piemonkey commented:
> https://github.com/lblod/ember-rdfa-editor/blob/09451449a9bcde555ea2c331c5672f2e7c552948/packages/ember-rdfa-editor/src/components/_private/relationship-editor/card.gts#L444
> gives type errors now, as the `@isTopLevel` argument is not passed.
> Alternatively we could also not pass the `isTopLevel` arg to the `configurable-rdfa-display` component, and already encode it in the generator function we pass to that component. (as the component itself doesn't really need that argument, and it is not super relevant for every usage of it)

Oops, was sure I changed that one too...

I thought about it and actually, I think it makes more sense to have `isTopLevel` passed as part of the whole context, as otherwise, you're right that we're passing things to the component that it doesn't directly care about. I've changed that over and fixed the type error too.
